File: src/main/java/ec/edu/grupo3/view/ConversionResponseView.java
package ec.edu.grupo3.view;

import jakarta.xml.bind.annotation.XmlAccessType;
import jakarta.xml.bind.annotation.XmlAccessorType;
import jakarta.xml.bind.annotation.XmlType;

@XmlAccessorType(XmlAccessType.FIELD)
@XmlType(name = "conversionResponse", propOrder = {"category", "fromUnit", "toUnit", "inputValue", "resultValue"})
public class ConversionResponseView {

    private String category;
    private String fromUnit;
    private String toUnit;
    private double inputValue;
    private double resultValue;

    public ConversionResponseView() {
    }

    public ConversionResponseView(String category, String fromUnit, String toUnit, double inputValue, double resultValue) {
        this.category = category;
        this.fromUnit = fromUnit;
        this.toUnit = toUnit;
        this.inputValue = inputValue;
        this.resultValue = resultValue;
    }

    public String getCategory() {
        return category;
    }

    public void setCategory(String category) {
        this.category = category;
    }

    public String getFromUnit() {
        return fromUnit;
    }

    public void setFromUnit(String fromUnit) {
        this.fromUnit = fromUnit;
    }

    public String getToUnit() {
        return toUnit;
    }

    public void setToUnit(String toUnit) {
        this.toUnit = toUnit;
    }

    public double getInputValue() {
        return inputValue;
    }

    public void setInputValue(double inputValue) {
        this.inputValue = inputValue;
    }

    public double getResultValue() {
        return resultValue;
    }

    public void setResultValue(double resultValue) {
        this.resultValue = resultValue;
    }
}

File: dotnet-wcf/ConversionApp/Views/ConversionResponseView.cs
using System.Runtime.Serialization;

namespace ConversionApp.Views
{
    [DataContract(Name = "conversionResponse", Namespace = "")]
    public class ConversionResponseView
    {
        [DataMember(Order = 1)]
        public string Category { get; set; }

        [DataMember(Order = 2)]
        public string FromUnit { get; set; }

        [DataMember(Order = 3)]
        public string ToUnit { get; set; }

        [DataMember(Order = 4)]
        public double InputValue { get; set; }

        [DataMember(Order = 5)]
        public double ResultValue { get; set; }

        public ConversionResponseView() { }

        public ConversionResponseView(string category, string fromUnit, string toUnit, double inputValue, double resultValue)
        {
            Category = category;
            FromUnit = fromUnit;
            ToUnit = toUnit;
            InputValue = inputValue;
            ResultValue = resultValue;
        }
    }
}
